lib/config.js

```diff
@@ -53,19 +58,13 @@ config.mongodb.connectOptions = {
   promoteBuffers: true,
 };
 config.mongodb.writeOptions = {
-  j: true,
-  w: 'majority',
-  // as mongo 4.2 this is no longer used by updateOne or updateMany
-  // FIXME remove this once all dependencies have been upgraded
-  multi: true,
+  writeConcern,
```

Comment:
If this is already set on `connectOptions` -- does it need to be set here as well? Or will `connectOptions` form the default that is used and we can omit it here?

Reply:
the answer to this is inconsistent at this point. These deprecation notices are brand new and mongodb's native node driver still stumbles on them all over the place. The defaults for all write options in the mongodb node driver as of this morning still used `w`, `j`, `fsync` in options and not in the newer option `writeConcern`. So basically, we can not rely on mongodb to satisfy their own deprecation in the current release of the driver.